<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminGuiClass</class>
 <widget class="QWidget" name="AdminGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminGui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(253, 240, 224)</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Britannic Bold</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(253, 240, 224)</string>
       </property>
       <property name="text">
        <string>Tutorials List</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="listWidget_tutorials">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(85, 85, 127);
border-color: rgb(253, 240, 224)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_title"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Britannic Bold</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Presenter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_presenter"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Britannic Bold</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Duration  in minutes</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_duration_minutes"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Britannic Bold</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Duration in seconds</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_duration_seconds"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Britannic Bold</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Number of likes</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_number_likes"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Britannic Bold</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Link</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_link"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>Britannic Bold</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Title tutorial to remove:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_remove_title"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Britannic Bold</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Title tutorial to update</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEdit_update_title"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_add">
         <property name="font">
          <font>
           <family>Britannic Bold</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(172, 177, 214)</string>
         </property>
         <property name="text">
          <string>ADD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_remove">
         <property name="font">
          <font>
           <family>Britannic Bold</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(172, 177, 214)</string>
         </property>
         <property name="text">
          <string>REMOVE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_update">
         <property name="font">
          <font>
           <family>Britannic Bold</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(172, 177, 214)</string>
         </property>
         <property name="text">
          <string>UPDATE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_exit">
         <property name="font">
          <font>
           <family>Britannic Bold</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(172, 177, 214)</string>
         </property>
         <property name="text">
          <string>EXIT</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
